python/lsst/pipe/tasks/detectAndMeasure.py

```diff
@@ -189,7 +189,6 @@ def run(self, exposure, exposureIdInfo, background=None, allowApCorr=True):
             self.deblend.run(
                 exposure = exposure,
```

Comment:
This doesn't need to be 3 lines any more: it'll should fit on a single line just fine.